docker/mysql-xtrabackup/Dockerfile

```diff
@@ -42,7 +42,7 @@ RUN set -ex; \
 	apt-key list > /dev/null
 
 ENV MYSQL_MAJOR 5.7
-ENV MYSQL_VERSION 5.7.27-1debian8
+ENV MYSQL_VERSION 5.7.28-1debian8
```

Comment:
hi @jesdynf ,
Let me know if you need me to rebuild this docker with this change (when Stephen brings it in) and push it to dockerhub.
thanks, -brady

Reply:
hi @bradymiller , don't have write access to this repo (for a good reason 😄)